lib/Parse/ParseSIL.cpp

```diff
@@ -884,13 +884,13 @@ bool SILParser::parseSILType(SILType &Result,
     bool walkToTypeReprPre(TypeRepr *T) {
       if (auto fnType = dyn_cast<FunctionTypeRepr>(T)) {
         if (auto generics = fnType->getGenericParams()) {
-          auto env = handleSILGenericParams(C, generics, SF);
+          auto env = handleSILGenericParams(C, generics, SF, /* FIXME */ nullptr);
```

Comment:
I'm not sure if `nullptr` is the best choice here, but I also can't see how to get anything better, given this is just inside `parseSILType` (similarly for the one below).